src/components/calcite-stepper-item/calcite-stepper-item.scss

```diff
@@ -142,30 +134,29 @@
   margin-right: initial;
 }
 :host([disabled]) {
-  opacity: var(--calcite-ui-opacity-disabled);
+  @apply opacity-disabled;
 }
 :host([disabled]),
 :host([disabled]) * {
-  cursor: not-allowed;
-  pointer-events: none;
+  @apply cursor-not-allowed pointer-events-auto;
 }
 
 :host([complete]) {
   // todo dark theme
   border-top-color: rgba($h-bb-060, 0.5);
```

Comment:
It would be cool if we could kill this function call.

Reply:
not sure I can do that

Reply:
lets add this to the list of todos

Reply:
added